api-schema/xcherry.yaml

```diff
@@ -237,6 +259,50 @@ components:
         - FAILED
         - TIMEOUT
         - TERMINATED
+    ListProcessExecutionsRequest:
+      type: object
+      required:
+        - namespace
+        - pageSize
+        - query
+      properties:
+        namespace:
+          type: string
+        pageSize:
+          type: integer
+          default: 100
+        nextPageToken:
+          type: string
+        query:
```

Comment:
What do you expect to use this field?

Reply:
similar to the implementation in cadence/temporal, this query is pretty much the WHERE clause in a sql query, to specify the process execution filter